src/main/java/bio/terra/workspace/service/job/JobService.java

```diff
@@ -253,32 +251,41 @@ public JobModel mapFlightStateToJobModel(FlightState flightState) {
       completedDate = flightState.getCompleted().get().toString();
     }
 
-    JobModel jobModel =
-        new JobModel()
+    JobReport jobReport =
+        new JobReport()
             .id(flightState.getFlightId())
             .description(description)
             .status(jobStatus)
             .statusCode(statusCode.value())
             .submitted(submittedDate)
-            .completed(completedDate);
+            .completed(completedDate)
+            .resultURL(resultUrlFromFlightState(flightState));
 
-    return jobModel;
+    return jobReport;
   }
 
-  private JobModel.StatusEnum getJobStatus(FlightStatus flightStatus) {
+  private String resultUrlFromFlightState(FlightState flightState) {
+    String relativePath =
+        flightState
+            .getInputParameters()
+            .get(JobMapKeys.RESULT_RELATIVE_PATH.getKeyName(), String.class);
+    return ingressConfig.getDomainName() + relativePath;
```

Comment:
Is it a safe assumption that the domainName configuration param has a trailing param?
Might want something like:
```
String domain = ingressConfig.getDomainName();
return domain.endsWith("/") ? domain + relativePath : domain + "/" + relativePath
```

Reply:
This makes the assumption that domainName does not have a trailing "/" and that `relativePath` does have a leading "/". I need to rethink that name a bit

Reply:
I think you could use the Path.of(domain, relativePath) and let it figure it out.